pyuvdata/uvcal/uvcal.py

```diff
@@ -1192,7 +1194,11 @@ def _check_freq_spacing(self, raise_errors=True):
         )
 
     def check(
-        self, check_extra=True, run_check_acceptability=True, check_freq_spacing=False
+        self,
+        check_extra=True,
+        run_check_acceptability=True,
+        check_freq_spacing=False,
+        astrometry_library=None,
```

Comment:
Part of me feels conflicted adding this to something like `check` -- I don't know how often it's called by users, but fundamentally each of the underlying astrometry libraries are _supposed_ to give answers to much better than the UVParameter tolerances. Is there potential the "red herrings" this might produce outweigh the benefit added by including it? Maybe that's a vote for just adding a disclaimer in the docstring rather than dropping it all together, but just my 2 cents...

Reply:
I think you're right, I don't think it's needed in the check given our tests. I'll take it out.